CodeReview/src/overlays/CodeReviewCommentOverlay.cpp

```diff
@@ -55,6 +54,8 @@ void CodeReviewCommentOverlay::updateGeometry(int availableWidth, int availableH
 {
 	Super::updateGeometry(availableWidth, availableHeight);
 	setPos(associatedItem()->mapToScene(offsetItemLocal_));
+
+	setScale(1.0/Visualization::VisualizationManager::instance().mainScene()->mainViewScalingFactor());
```

Comment:
I'm not entirely happy with this change. Could you describe what the problem is?

Also do you think this could be fixed, instead of here, by making your structure aware zoom check for the ItemIgnoresTransformations flag and adjust its behavior appropriately? If so, please experiment with that and let me know.